portal-web/test/functional/com/liferay/portalweb/macros/fragment/FragmentsAdminNavigator.macro

```diff
@@ -42,7 +42,7 @@ definition {
 			Button.click(button = "New");
 		}
 
-		Button.click(button = "Import");
+		Button.clickDropdownItem(item = "Import");
```

Comment:
We have DropdownMenuItem.click already available. I'll revert this and forward with new commit since the dropdown isn't technically a button.

Reply:
Other than that looks good!